api/v1/storagecluster_types.go

```diff
@@ -137,11 +148,42 @@ type ManageCephObjectStoreUsers struct {
 	ReconcileStrategy string `json:"reconcileStrategy,omitempty"`
 }
 
+// ExternalStorageMode specifies a mode of the external storage
+type ExternalStorageMode string
+
+const (
+	// ModeOcsToOcs specifies a "ocs-to-ocs" mode of the external storage
+	ModeOcsToOcs ExternalStorageMode = "ocs-to-ocs"
+
+	// ModeStandaloneCeph specifies a "standalone-ceph" mode of the external storage
+	ModeStandaloneCeph ExternalStorageMode = "standalone-ceph"
```

Comment:
would this mean existing users have to update their spec to have `standalone-ceph` mode?

Reply:
No, the spec will default to using the normal external mode if not specified.